src/v2i-hub/TelematicBridgePlugin/src/TelematicBridgePlugin.cpp

```diff
@@ -4,6 +4,8 @@ namespace TelematicBridge
 {
     TelematicBridgePlugin::TelematicBridgePlugin(const string &name) : PluginClient(name)
     {
+        _telematicUnitPtr = make_shared<TelematicUnit>();
```

Comment:
Can this be a unique pt?

Reply:
Is there a reason for this change?

Reply:
This point is shared with this plugin. I cannot find a case where users will accidentally change the lifecycle of this pointer.

Reply:
If the plugin has a reference to this object but the telematics bridge  is no longer in scope, the telematics bridge will still not get deleted because the plugin holds a reference to this pointer which will prevent the pointer from getting deleted

Reply:
Why does the plugin need a reference to this pointer?

Reply:
So that the plugin can use this pointer to send data into the telematic tool. It does not make sense for this plugin to subscribe messages but not forward them into the telematic tool.

Reply:
@paulbourelly999  Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/567/commits/040a99b1b4e251b04731c572cbc2da450bd0bcd0